Fix gradle lint

Currently, lint reports the following issues:
```
  Errors found:
  
  /home/marcin/proj/android/QDict/qDict/src/main/java/com/mmt/app/SystemBarTintManager.java:104: Error: Expected resource of type styleable [ResourceType]
                  mNavBarAvailable = a.getBoolean(1, false);
                                                  ~
  /home/marcin/proj/android/QDict/qDict/src/main/java/com/mmt/widget/DragSortListView.java:2805: Error: Call requires API level 21 (current min is 17): android.widget.AbsListView#setSelectionFromTop [NewApi]
              setSelectionFromTop(movePos, top - padTop);
              ~~~~~~~~~~~~~~~~~~~
  /home/marcin/proj/android/QDict/qDict/src/main/java/com/annie/dictionary/standout/StandOutWindow.java:583: Error: Must be one or more of: PendingIntent.FLAG_ONE_SHOT, PendingIntent.FLAG_NO_CREATE, PendingIntent.FLAG_CANCEL_CURRENT, PendingIntent.FLAG_UPDATE_CURRENT, PendingIntent.FLAG_IMMUTABLE, Intent.FILL_IN_ACTION, Intent.FILL_IN_DATA, Intent.FILL_IN_CATEGORIES, Intent.FILL_IN_COMPONENT, Intent.FILL_IN_PACKAGE, Intent.FILL_IN_SOURCE_BOUNDS, Intent.FILL_IN_SELECTOR, Intent.FILL_IN_CLIP_DATA [WrongConstant]
                      Intent.FLAG_ACTIVITY_NEW_TASK);
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/qDict/build.gradle
+++ b/qDict/build.gradle
@@ -6,7 +6,7 @@ android {
     compileSdkVersion 29
     defaultConfig {
         applicationId "com.annie.dictionary.fork"
-        minSdkVersion 17
+        minSdkVersion 21
         targetSdkVersion 29
         versionCode 20210415
         externalNativeBuild {

--- a/qDict/src/main/java/com/annie/dictionary/standout/StandOutWindow.java
+++ b/qDict/src/main/java/com/annie/dictionary/standout/StandOutWindow.java
@@ -578,9 +578,9 @@ public abstract class StandOutWindow extends Service {
         PendingIntent contentIntent = null;
 
         if (notificationIntent != null) {
+            // flag updates existing persistent notification
             contentIntent = PendingIntent.getActivity(this, 0, notificationIntent,
-                    // flag updates existing persistent notification
-                    Intent.FLAG_ACTIVITY_NEW_TASK);
+                PendingIntent.FLAG_UPDATE_CURRENT);
         }
         NotificationCompat.Builder compatBuilder = new NotificationCompat.Builder(this);
         compatBuilder.setSmallIcon(icon).setContentTitle(contentTitle).setContentText(contentText)

--- a/qDict/src/main/java/com/mmt/app/SystemBarTintManager.java
+++ b/qDict/src/main/java/com/mmt/app/SystemBarTintManager.java
@@ -87,6 +87,7 @@ public class SystemBarTintManager {
      *
      * @param activity The host activity.
      */
+    @SuppressWarnings("ResourceType")
     @TargetApi(19)
     public SystemBarTintManager(Activity activity) {
 

